### ISSUE TITLE
[Feature request] Pause the timer and save it

### ISSUE THREAD

**Issue by shane1595042264** [LiveSplit/LiveSplit#2050]
As the title said, I want to save the pause time and resume the run next time I open it.
Some of you may say I can use set the time starts at a certain time so I can resume it. I mean it is a good alternative for single split, but I susually like using multiple splits.
So when I have multiple splits, and I set the timer starts at the latest split time, so that I hoped I can resume my run on that certain split. But unfortunately, when next time I run it, it just starts over at the first split again, and cover the first split with the latest split time......

For example, I have 3 seconds at the 1st split, 10 minutes at the 5th split. So I pause it, and set the timer starts at 10 minutes. When I press start the timer after that, the first split becomes 10 minutes and continues. 

In short, I really hope you smart guys could add this pause and save feature. I don't know if that's tricky since I am not programmer. But I suggest it would be super duper convenient if I can save the pause time by clicking "save the splits" instead of saving it as my PB and reseting the whole timer......

Appreciate all you guys' work, good luck!

**Comment by benshemmeld** [comment]
@shane1595042264 I think your idea has been implemented in this pull request: https://github.com/LiveSplit/LiveSplit/pull/1774

**Comment by Shotnex4** [comment]
Hey everyone, I also missed a feature to save my splits / timer mid run, close LiveSplit and continue later.
I developed a LiveSplit Component which you can use to save your progress mid run to a file. With that you can close LiveSplit and even shutdown your PC. Just import the file when you are back and you can continue:
https://github.com/Shotnex4/LiveSplit.TimeAttackPause
